install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -625,24 +631,27 @@ metadata:
 spec:
   match: context.protocol == "tcp"
   actions:
-  - handler: handler.prometheus
+  - handler: prometheus
     instances:
     - tcpbytesent.metric
     - tcpbytereceived.metric
 ---
 
 apiVersion: "config.istio.io/v1alpha2"
-kind: kubernetesenv
+kind: handler
 metadata:
-  name: handler
+  name: kubernetesenv
   namespace: {{ .Release.Namespace }}
 spec:
-  # when running from mixer root, use the following config after adding a
-  # symbolic link to a kubernetes config file via:
-  #
-  # $ ln -s ~/.kube/config mixer/adapter/kubernetes/kubeconfig
-  #
-  # kubeconfig_path: "mixer/adapter/kubernetes/kubeconfig"
+  compiledAdapter: kubernetesenv
+  adapter: kubernetesenv
```

Comment:
Per documentation, this refers to OOP adapter. `compiledAdapter` is required for in-process adapters.

Reply:
They are both listed as `Required`. I will try removing `adapter` and see what happens.

Reply:
It all seems to work without `adapter`. Removed here and everywhere else now.